Makefile

```
@@ -69,22 +72,36 @@ dev.editable-envs:  ## Copy env files outside the docker containers so it's edit
 	@make studio-restart
 
 dev.up: | check-memory ## Bring up all services with host volumes
-	docker-compose -f docker-compose.yml -f docker-compose-host.yml -f docker-compose-themes.yml -f docker-compose-watchers.yml up -d
+	docker-compose -f docker-compose.yml -f docker-compose-host.yml up -d
+
+dev.nfs.setup:  ## set's up an nfs server on the /Users folder, allowing nfs mounting on docker
+	./setup_native_nfs_docker_osx.sh
+
+dev.nfs.up: | check-memory ## Bring up all services with host volumes
+	docker-compose -f docker-composedraak_dev_routere.yml -f docker-compose-host-nfs.yml up -d
```

```suggestion
	docker-compose -f docker-compose.yml -f docker-compose-host-nfs.yml up -d
```


I couldn't find where `composedraak_dev_routere` is.